Rework using HTTP proxy with compute resources (#3994)
theforeman/foreman-documentation: guides/common/modules/con_using-an-http-proxy-to-communicate-with-compute-resources.adoc

:_mod-docs-content-type: CONCEPT

[id="using-an-http-proxy-to-communicate-with-{compute-resource-context}"]
= Using an HTTP proxy to communicate with {CRname}

> If your {CRname} requires an HTTP proxy to communicate with {Project}, you have to setup and select an HTTP proxy in {Project}.

Reviewer: Wondering if this seems better:

```suggestion
If your {CRname} requires an HTTP proxy to communicate with {Project}, you have to setup and select the HTTP proxy in {Project}.
```

Author: Applied.

> However, if you configure an HTTP proxy for {Project} in *Administer* > *Settings*, and then add another HTTP proxy for your compute resource, the HTTP proxy that you define in *Administer* > *Settings* takes precedence.

Reviewer: Not sure about this, but just a possible alternative as the use of the adverb `then` seems to imply that the sequence of events is important here:

```suggestion
Note that if you have configured an HTTP proxy for {Project} in *Administer* > *Settings*, it takes precedence over the HTTP proxy for your compute resource.
```

Feel free to ignore of this suggestion, or to change it as you see fit. (Unless this happens only if the *Administer* > *Settings* HTTP proxy has been set before adding compute resource HTTP proxy.)

Author: Applied and removed the leading "Note that".
